### PR TITLE
renaming db name and role to fit convention

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -140,17 +133,7 @@ jobs:
...
-            echo "    pypi " >> ~/.pypirc
-            echo >> ~/.pypirc
-            echo "[pypi]" >> ~/.pypirc
-            echo "repository:https://upload.pypi.org/legacy/" >> ~/.pypirc
-            echo "username:$PYPI_USERNAME" >> ~/.pypirc
-            echo "password:$PYPI_PASSWORD" >> ~/.pypirc
-            echo >> ~/.pypirc
-            python setup.py register -r pypi
-            python setup.py sdist upload -r pypi
+            echo "Not publishing package!"
```
Comment (GabeLambda) [inline .circleci/config.yml]: ??

File: README.md [hunk 2]
```diff
@@ -133,3 +133,9 @@ Example:
         COMPLETED = event_info(
             follows=event("STARTED"),
         )
+
+
+### Running tests
+
+'microcosm-evensource' required a db user named 'microcosm_eventsource' and a db named 'microcosm_eventsource_test_db' - create both and run tests using `nosetests`
```
Comment (GabeLambda) [inline README.md]: typo: eventsource

I'd also just put in the exact commands to copy-paste in:
```
        createuser microcosm_eventsource
        createdb -O microcosm_eventsource microcosm_eventsource_db
        createdb -O microcosm_eventsource microcosm_eventsource_test_db
```

File: .circleci/config.yml (hunk 1 above)
Comment (unorthodoxgeek) [reply .circleci/config.yml]: this is the autogenerated output... that's odd.

Comment (KensoDev) [reply .circleci/config.yml]: The `pypi` key changed. 

you need to add:

```
"pypi": {
  "repository": "pypi"
}
```

**Pull request closed without merging**